src/Routes/RosPage/RosPage.js

```diff
@@ -310,9 +341,31 @@ class RosPage extends React.Component {
                                     }
                                 ]
                             }}
+                            exportConfig={{
+                                isDisabled: this.state.disableExport,
+                                extraItems: [<Button
+                                    key='pdf-download-button' variant='plain'
+                                    onClick={() => this.setExportSystemsPDF(true)}>
+                                        Export as PDF
+                                </Button>],
+                                ouiaId: 'export',
+                                onSelect: (_event, fileType) => this.onExportOptionSelect(fileType)
+                            }}
                             onExpandClick={(_e, _i, isOpen, { id }) => this.props.expandRow(id, isOpen, 'EXPAND_ROW')}
                         >
                         </InventoryTable>
+                        {exportSystemsPDF &&
+                            <DownloadSystemsPDFReport
```

Comment:
Hello @r14chandra yes that's correct. Since it is a dropdown component I tried using <DropdownItem /> from patternfly but it was giving error. Hence I used a button component but the hoving style went away. I will check if there is an easy fix for same.
I had asked this question in the platform ui channel but didn't get any response for same. [Here](https://ansible.slack.com/archives/C023VGW21NU/p1648037908608969) is the link to conversation.

Reply:
created Jira card to fix this issue separately https://issues.redhat.com/browse/RHIROS-618